radis/db/classes.py

```diff
@@ -217,6 +219,38 @@
 """ str: list of [HITRAN-2020]_ molecules. """
 
 
+def is_atom(species):
+    if "_I" in species:
+        return True
+    else:
+        return False
+
+
+def to_conventional_name(species):
+
+    # If the charge is positive
+    if "+" in species:
+        # Count the number of positive charges
+        charge = species.count("+") + 1  # Add one to convert to conventional notation
+        # Get the element of the species without the charge
+        element_name = species.split("+")[0]
+        # Convert the charge to roman notation
+        charge_in_roman = {1: "I", 2: "II", 3: "III", 4: "IV", 5: "V"}.get(charge, "")
+        return f"{element_name}_{charge_in_roman}"
+
+    # If species is an element
+    else:
+        try:
+            atomic_symbol = species.split("_")[0]
+            el = getattr(periodictable, atomic_symbol)
+            if el.symbol == species:  # check if symbols match
+                return f"{species}_I"
```

Comment:
I don't think that we need this test with the package periodictable.

Reply:
For checking whether the species in that form is an atom, in which case append '_I' to put it in spectroscopic notation, or not in which case return as is.